source/tutorials/metadata.rst

```diff
@@ -0,0 +1,141 @@
+
+To get started with understanding artifacts as metadata, first download an example artifact:
+
+.. download::
+   :url: https://docs.qiime2.org/2017.6/data/tutorials/moving-pictures/core-metrics-results/faith_pd_vector.qza
+   :saveas: faith_pd_vector.qza
+
+To view this artifact as metadata, simply pass it in to any method or visualizer that expects to see metadata (e.g. ``metadata tabulate`` or ``emperor plot``):
+
+.. command-block::
+   qiime metadata tabulate \
+     --m-input-file faith_pd_vector.qza \
+     --o-visualization tabulated-faith-pd-metadata.qzv
+
+.. question::
+   What is the largest value of Faith's PD? What is the smallest? Hint: use the column sorting functions to assist with this query.
+
+Combining metadata
+------------------
+
+Because metadata can come from many different sources, QIIME 2 supports metadata merging when running commands. Building upon the examples above, simply passing ``--m-input-file`` multiple times will combine the metadata in the specified files:
+
+.. command-block::
+   qiime metadata tabulate \
+     --m-input-file sample-metadata.tsv \
+     --m-input-file faith_pd_vector.qza \
+     --o-visualization tabulated-combined-metadata.qzv
+
+Merging is based on the identifiers specified in the first column of each metadata file (i.e. sample IDs or feature IDs). The resulting metadata after the merge will contain the intersection of the identifiers across all of the specified files. Merging does not modify any of the input files.
+
+.. question::
+   Modify the command above to merge the `evenness vector`_ of ``SampleData[AlphaDiversity]`` after the Faith's PD vector. What happens when merging the three artifacts? How many columns are present in the resulting metadata visualization? How many of those columns represent the sample IDs? How many of those columns represent ``SampleData[AlphaDiversity]`` metrics?
+
+Metadata merging is supported anywhere that metadata is accepted in QIIME 2. For example, it might be interesting to color an Emperor plot based on the study metadata, or the sample alpha diversities. This can be accomplished by providing both the sample metadata mapping file *and* the ``SampleData[AlphaDiversity]`` artifact:
+
+.. download::
+   :url: https://docs.qiime2.org/2017.6/data/tutorials/moving-pictures/core-metrics-results/unweighted_unifrac_pcoa_results.qza
+   :saveas: unweighted_unifrac_pcoa_results.qza
+
+.. command-block::
+   qiime emperor plot \
+     --i-pcoa unweighted_unifrac_pcoa_results.qza \
+     --m-metadata-file sample-metadata.tsv \
+     --m-metadata-file faith_pd_vector.qza \
+     --o-visualization unweighted-unifrac-emperor-with-alpha.qzv
+
+.. question::
+   What happens to the visualization if the order of the metadata files is reversed in the above ``emperor plot``? What about in the ``metadata tabulate`` example above?
```

Comment:
Is the answer to the first question "nothing changes"? I'm not sure I understand what the first question is driving at since Emperor does its own sorting of categories to color by.